lua/entities/gmod_tardis/modules/cl_classic_doors.lua

```diff
@@ -14,4 +14,23 @@ ENT:AddHook("ShouldDrawPart", "classic_doors_exterior_door", function(self, part
 	then
 		return false
 	end
-end)
\ No newline at end of file
+end)
+
+ENT:AddHook("PlayerEnter", "classic_doors_intdoor_sound", function(self,ply,notp)
+	if not IsValid(self.interior) then return end
+	if self.metadata.EnableClassicDoors ~= true then return end
+	if self.metadata.NoSoundOnEnter == true then return end
+
+	local intdoor = TARDIS:GetPart(self.interior, "intdoor")
+	if not IsValid(intdoor) then return end
+
+	local door_sounds = self.metadata.Interior.Sounds.Door
+	if not door_sounds then return end
+
+	local door_sound = self:GetData("doorstatereal") and door_sounds.open or door_sounds.close
+	if not door_sound then return end
+
+	if intdoor.IntDoorPos ~= nil and intdoor.IntDoorPos ~= 0 and intdoor.IntDoorPos ~= 1 then
+		sound.Play(door_sound, self.interior:LocalToWorld(Vector(0,0,0)))
```

Comment:
Any reason why we have LocalToWorld(Vector(0,0,0)) vs just GetPos()? In this context they are identical, also should the door sound be coming from the centre of the interior?

Reply:
Left it there accidentally
I've now changed it to use fallback coordinates instead